tests/integration/gcp.py

```diff
@@ -519,21 +514,30 @@ def create_vm(self):
             },
         }
 
-        operation = self._compute.instances().insert(project=self.project, zone=self.zone, body=config).execute()
+        if self.config['secureboot']:
+            shielded_instance_config = {
+                "shielded_instance_config": {
+                    "enable_secure_boot": True,
+                    "enable_integrity_monitoring": True,
+                    "enable_vtpm": True,
+                }
+            }
+            config.update(shielded_instance_config)
+
+        operation = self._compute_instances.insert(project=self.project, zone=self.zone, instance_resource=config)
         self._gcp_wait_for_operation(operation)
-        url = operation.get("targetLink")
+        url = self._compute_instances.get(project=self.project, zone=self.zone, instance=name).self_link
         self._instance_name = self._get_resource_name(url)
```

Comment:
Does `_get_resource_name` return another name than the already available variable `name` in this function? If not, we may could remove this function here as well.